doc-src/SASS_REFERENCE.md

```diff
@@ -811,6 +812,13 @@ color interpolated into a selector becomes invalid syntax when
 compressed. To avoid this, always quote named colors if they are meant
 to be used in the construction of a selector.
 
+#### Callables
+
+A callable value is returned by `function-reference($function-name)`.
+The callable value can be passed to `call($callable, $args...)` and the
```

Comment:
We definitely shouldn't position `call()` as able to call any callable—we won't allow mixins to be passed to it.